src/test/java11/io/opentracing/contrib/jfrtracer/JfrTracerTest.java

```diff
@@ -62,6 +62,14 @@ public void basicEvent() throws IOException {
 				tracer.buildSpan("test span").start().finish();
 
 				recording.dump(output);
+				recording.stop();
+			}
+
+			//to be removed after test are fixed due to the concurrency
```

Comment:
Comments start with capital letter. Also, instead of catching interrupted exception everywhere, just throw it or add a helper method.

Reply:
I've add throw InterruptedException